index.html

```diff
@@ -226,14 +240,11 @@ <h2 class="form-text">
                         <a href="#"><img src="images/Vector3.png" alt="social media icons"></a>
                     </li>
                 </ul>
-                   </form>
+                 
                    <a class="my" href="#">jemutaikiplagat@gmail.com</a>
-            </div>
-        </section> 
+
         
-    </footer>
-    
-    
+    </footer> 
```

Comment:
- [ ] [OPTIONAL] There is a very huge white space below the footer section. This makes your website not user-friendly. Please make sure you remove it because people will keep on scrolling believing that there are other contents.